Mesh_3/include/CGAL/Mesh_polyhedron_3.h

```diff
@@ -215,6 +217,117 @@ struct Mesh_polyhedron_3
   typedef type Type;
 };
 
+
+template <typename Gt, typename Patch_id>
+struct Patch_id_pmap {
```

Comment:
Can't we just ignore it for now and wait for #2023, which will apply all the modifications in the right order, instead of trying to cherry pick commits that manipulate properties that have been deeply reworked in #2023 but not here, creating conflicts that I do not understand enough to fix them ?